sycl/test-e2e/Matrix/get_coordinate_ops_impl.hpp

```diff
@@ -124,11 +122,7 @@ void test_get_coord_op() {
   TResult sum_cols[Cols] = {0};
   TResult sum_cols_ref[Cols] = {0};
 
-  for (int i = 0; i < Rows; i++) {
-    for (int j = 0; j < Cols; j++) {
-      M[i][j] = i + j;
-    }
-  }
+  matrix_fill(Rows, Cols, (T *)M, [](int i, int j) { return T(1) * (i + j); });
```

Comment:
```suggestion
  matrix_fill(Rows, Cols, (T *)M, [](int i, int j) { return T(i + j); });
```

Reply:
Thanks for the suggestion! I have fix this :)